pincer/middleware/ready.py

```diff
@@ -23,7 +23,7 @@ async def on_ready_middleware(self, payload: GatewayDispatch):
         The data received from the ready event.
     """
     self.bot = User.from_dict(
-        {"_client": self, "_http": self.http,  **payload.data.get("user")}
+        {"_client": self, "_http": self.http, **payload.data.get("user")}

```

Comment:
If `payload.data.get("user")` returns None, then this would cause an error.

Reply:
Resolved, now it will throw a `InvalidPayload`